byline-manager.php

```diff
@@ -61,3 +61,5 @@ function validate_path( string $path ) : bool {
 
```

Comment:
is the `package-lock.json` update intentional?

Reply:
Okay, in the main branch, if I run `npm audit` those packages are showing as having `moderate` to `high` vulnerabilities. I'll keep this file change.

Reply:
Cool!